modules/server/src/main/scala/navigate/server/Systems.scala

```diff
@@ -41,12 +41,27 @@ object Systems {
       if (conf.systemControl.tcs === ControlStrategy.FullControl)
         for {
           tcs <- TcsEpicsSystem.build(epicsSrv, tops)
-          p1  <- WfsEpicsSystem.build(epicsSrv, "PWFS1", readTop(tops, "pwfs1".refined))
-          p2  <- WfsEpicsSystem.build(epicsSrv, "PWFS2", readTop(tops, "pwfs2".refined))
-          oi  <- WfsEpicsSystem.build(epicsSrv,
-                                      "OIWFS",
-                                      readTop(tops, "oiwfs".refined),
-                                      "dc:initSigInit.MARK".refined
+          p1  <- WfsEpicsSystem.build(
+                   epicsSrv,
+                   "PWFS1",
+                   readTop(tops, "pwfs1".refined),
+                   "dc:fgDiag6P1.VALH".refined,
```

Comment:
No `top` for these?

Reply:
They are relative to the `top` given in the previous parameter.